pykickstart/parser.py

```diff
@@ -189,7 +189,7 @@ def __init__(self, *args, **kwargs):
            :keyword path: The path where the certificate should be stored.
         """
         KickstartObject.__init__(self, *args, **kwargs)
-        self.cert = "\n".join(kwargs.get("cert", []))
+        self.cert = "".join(kwargs.get("cert", [])).removesuffix("\n")
```

Comment:
This requires python 3.9 which is fine unless we need to backport to RHEL 8. I've modified the github tests in #518 so that we test on 3.9 and later.

Reply:
There are only tickets for rhel 9 and rhel 10 as of now but maybe I'd rather stay on safer side and use something supported.

Reply:
Updated.